services/resource-service/stub/src/main/proto/ResourceService.proto

```diff
@@ -80,6 +82,10 @@ message SCPResourceDeleteRequest {
     string resourceId = 1;
 }
 
+message SCPResourcesGetRequest {
+    string storageId = 1;
```

Comment:
Why do you need a storage id?

Reply:
@DImuthuUpe storageId is to filter the resources in a particular storage unit. Correct me if my understanding is wrong. We have different storage types like S3, SCP, Azure. There can be multiple storage units of each type eg remote-ssh-storage1, remote-ssh-storage. And within each storage unit there are resources which user can transfer from one storage unit to another. In the admin portal navigation would work as follows: User views all storage units available, selects a storage unit to view all the resources within. To request for resources within a storage unit, we need an storage identifier. I have only added this for SCP resources request since we only have SCP storages implemented.